src/test/java/com/softserve/academy/spaced/repetition/controller/CardControllerTest.java

```diff
@@ -68,14 +68,14 @@ public void getCardById() throws Exception {
                         "}"));
     }
 
-    @Test
-    public void getCardByWrongId() throws Exception {
-        when(cardService.getCard(eq(0L))).thenReturn(createCard());
-        mockMvc.perform(get("/api/card/{cardId}", 1L)
-                .accept(MediaType.APPLICATION_JSON)
-                .contentType(MediaType.APPLICATION_JSON))
-                .andExpect(status().isConflict());
-    }
+//    @Test
```

Comment:
Sonar: Commented code should be removed. If you ever need it, you will find it in Version Control System (Git).

Reply:
ok I'll fix.